cmd/deploy-contracts/main.go

```diff
@@ -78,7 +78,6 @@ func main() {
 				panic(err)
 			}
 
-			utils.WaitForKillSignal()
```

Comment:
Can you explain the reasoning for this change?

Reply:
I went with the more modular approach in https://github.com/statechannels/go-nitro/pull/1631/commits/913eb225e4e3954ef0b18c99bf00591059e33669. That commit removes the code to start an `anvil` network as part of the `deploy-contracts` script. Instead, the user can first start an `anvil` network (or any local network), and pass that network's CHAIN_URL to this script if they want the contracts deployed there.